test-e2e/assertions.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2015 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import { expect, Page } from '@playwright/test';
+import { ElectronApplication } from 'playwright';
+
+const getTitleOfWindow = (app: ElectronApplication, page: Page) =>
+    new Promise(resolve =>
+        app
+            .browserWindow(page)
+            .then(browserWindow => browserWindow.getProperty('title'))
+            .then(property => resolve(property.jsonValue()))
+    );
+
+export const checkTitleOfWindow = async (
+    app: ElectronApplication,
+    titleString: string
+) => {
+    const window = await app.firstWindow();
+    const title = await getTitleOfWindow(app, window);
+    expect(title).toContain(titleString);
+};
+
+export const checkTitleOfSecondWindow = async (
```

Comment:
This should rather be named `checkTitleOfLastWindow` or something similar since we are checking the window at the end of the list, not necessarily the second.